scheduler/src/cook/mesos/dru.clj

```diff
@@ -99,20 +100,18 @@
 
 (defn sorted-task-cumulative-gpu-score-pairs
   "Takes a sorted seq of task entities and the gpu divisor, returns a list of [task cumulative-gpus], preserving the same order of input tasks"
-  [user->dru-divisors user->sorted-running-task-ents]
+  [user->dru-divisors pool-name user->sorted-running-task-ents]
```

Comment:
Is `pool-name` used here?

Reply:
No, but this is the same "interface" as `sorted-task-scored-task-pairs` so I wanted to keep the arguments the same.